cmd/kube-proxy/app/server.go

```diff
@@ -383,7 +386,7 @@ type ProxyServer struct {
 
 // createClients creates a kube client and an event client from the given config and masterOverride.
 // TODO remove masterOverride when CLI flags are removed.
-func createClients(config componentconfig.ClientConnectionConfiguration, masterOverride string) (clientset.Interface, v1core.EventsGetter, error) {
+func createClients(config componentconfig.ClientConnectionConfiguration, masterOverride string) (clientset.Interface, v1core.CoreV1Interface, error) {
```

Comment:
I tried using api.Node instead of v1.Node when creating the TwoWayMerge which ended up with a 200 and nothing changed. The api.Node doesn't have any of the json annotation so the patch is incorrect. I am open to other suggestions.